src/components/Section/ProposalVotingSectionTestCases.tsx

```diff
@@ -0,0 +1,190 @@
+import { TestData } from '../../hooks/useVotingSectionTestData'
+
```

Comment:
This file (ProposalVotingSectionTestCases.tsx) can be .ts as it is not returning tsx.